src/hotspot/share/c1/c1_Compiler.cpp

```diff
@@ -222,11 +222,9 @@ bool Compiler::is_intrinsic_supported(const methodHandle& method) {
 #ifdef JFR_HAVE_INTRINSICS
   case vmIntrinsics::_counterTime:
   case vmIntrinsics::_getEventWriter:
-#if defined(_LP64) || !defined(TRACE_ID_SHIFT)
-  case vmIntrinsics::_getClassId:
+  // TODO: temporarily not implement getClassId in c1
```

Comment:
I think we need not worry about an intrinsic for C1, so please remove.

Reply:
removed.